tests/test_arr_venue_v2.py

```diff
@@ -5955,7 +5955,7 @@ def test_commitment_venue(self, client, test_client, openreview_client, helpers)
         notes = pc_client_v2.get_notes(invitation='aclweb.org/ACL/ARR/2023/August/-/Submission', number=3)
         assert len(notes) == 0
 
-        openreview.arr.ARR.process_commitment_venue(openreview_client, 'aclweb.org/ACL/2024/Workshop/C3NLP_ARR_Commitment')
+        openreview.arr.ARR.process_commitment_venue(openreview_client, 'aclweb.org/ACL/2024/Workshop/C3NLP_ARR_Commitment', get_previous_url_submission=True, identity_visibility=True)
```

Comment:
I would like you verify the changes are working here. There must be an ARR august submission that has a previous url, you need to assert that `aclweb.org/ACL/2024/Workshop/C3NLP_ARR_Commitment` has access to the previous submission and the identity of the reviewers ACs.